src/legacy/hooks/usePriceImpact/useQuoteAndSwap.ts

```diff
@@ -55,7 +55,7 @@ export function useCalculateQuote(params: GetQuoteParams) {
   } = params
   const { chainId: preChain } = useWalletInfo()
   const { account } = useWalletInfo()
-  const isEthFlow = useIsEthFlow()
+  const isEthFlow = useIsEoaEthFlow()
```

Comment:
if you decided to change the const from my last comment, change here too, otherwise ignore